kitchen.yml

```diff
@@ -92,6 +92,28 @@ suites:
         ubuntu_iptables: true
         redhat7_iptables: true
 
+  - name: nftables
```

Comment:
You might want to just do an includes here.

Reply:
Thanks for the hint.